### PR TITLE
Move benchmark out of protoc_plugin

### PR DESCRIPTION
**Pull request by sigurdm**

### CODE REVIEW

**Comment by sigurdm** [comment]
Should fix https://github.com/dart-lang/protobuf/issues/176

File: api_benchmark/lib/benchmarks/index.dart [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 library protoc.benchmark.index;
 
-import "../benchmark.dart";
+import '../benchmark.dart';
```
Comment (szakarias) [inline api_benchmark/lib/benchmarks/index.dart]: Correct all to single pling or none.

File: api_benchmark/lib/generated/benchmark.pb.dart [hunk 2]
```diff
@@ -0,0 +1,492 @@
+///
```
Comment (szakarias) [inline api_benchmark/lib/generated/benchmark.pb.dart]: No check in of generated files.

File: api_benchmark/lib/vm.dart [hunk 3]
```diff
@@ -46,7 +46,7 @@ runSuiteInVM(pb.Suite suite) async {
   var tmpFile = new File("$outFile.tmp");
   await tmpFile.writeAsString(encodeReport(lastReport));
   await tmpFile.rename(outFile);
-  print("\nWrote result to benchmark/data/${latestVMReportName}");
+  print("\nWrote result to ${dataDir}");
```
Comment (szakarias) [inline api_benchmark/lib/vm.dart]: outfile.

File: protoc_plugin/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,9 @@
+## 14.0.1
+
+* Remove the benchmark from the protoc_package. (it can be found in
```
Comment (szakarias) [inline protoc_plugin/CHANGELOG.md]: nit: it -> It

File: api_benchmark/lib/benchmarks/index.dart (hunk 1 above)
Comment (sigurdm) [reply api_benchmark/lib/benchmarks/index.dart]: I dropped it

File: api_benchmark/lib/generated/benchmark.pb.dart (hunk 2 above)
Comment (sigurdm) [reply api_benchmark/lib/generated/benchmark.pb.dart]: Done

File: api_benchmark/lib/vm.dart (hunk 3 above)
Comment (sigurdm) [reply api_benchmark/lib/vm.dart]: done

File: protoc_plugin/CHANGELOG.md (hunk 4 above)
Comment (sigurdm) [reply protoc_plugin/CHANGELOG.md]: done

**Pull request merged**